math/mathcore/inc/Math/IFunction.h

```diff
@@ -54,6 +54,8 @@ namespace ROOT {
           Derived classes must implement the pure private virtual method DoEval(const double *) for the
           function evaluation in addition to NDim() and Clone().
 
+          @see ROOT::Math::Functor is an ppropriate way to use one's own function with the fitting framework.
```

Comment:
```suggestion
          @see ROOT::Math::Functor is the recommended way to use one's own function with the fitting framework.
```
Typo, and maybe say "recommended" instead of "appropriate" (which is clearer IMHO)